sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -232,6 +232,13 @@ public class AADOAuth2LoginConditionalPolicyConfigSample extends WebSecurityConf
     }
 }
 ```
+### Customize scopes in authorize requests
+
+By default, `azure-spring-boot-starter-active-directory` configures scopes of `openid`, `profile` and `https://graph.microsoft.com/user.read` to implement OpenID Connect protocol and access of Microsoft Graph API. For customization of scope, developers need to configure in the `application.properties`:
+```yaml
```

Comment:
nit: this is not yaml format

Reply:
thanks for reminding, I will fix this in another pr